daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -56,6 +57,7 @@ General guidance on upgrading can be found for [self hosted mode]({{<ref self-ho
 | 1.1.0 to 1.1.2           |                   N/A |                    1.2.2 |
 |                          |                 1.2.2 |                    1.3.0 |
```

Comment:
Need additional rows here for updating from 1.1.0 to 1.4.0 (through v1.3)
Same for the the upgrading from 1.0.0

Reply:
Done. Please review again, thanks!